runtime/common/src/lib.rs

```diff
@@ -111,6 +113,27 @@ parameter_types! {
 	pub const MultiPaymentCurrencySetFee: Pays = Pays::Yes;
 }
 
+// pallet proxy
+/// The type used to represent the kinds of proxying allowed.
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, RuntimeDebug, MaxEncodedLen, TypeInfo)]
+pub enum ProxyType {
+	Any,
+}
+impl Default for ProxyType {
+	fn default() -> Self {
+		Self::Any
+	}
+}
+
+parameter_types! {
+	pub ProxyDepositBase: Balance = 4 * DOLLARS + 480 * MILLICENTS;
```

Comment:
is there some logic behind these amounts?

Reply:
This value should be equivalent to what Acala uses.

Reply:
equivalent, meaning that it takes into account that Snek supply is 100x Acala?

Reply:
I used expected dollar values. https://github.com/AcalaNetwork/Acala/blob/ec911c5db90aef7cce00d3143415886369860216/runtime/acala/src/constants.rs#L37

Reply:
> equivalent, meaning that it takes into account that Snek supply is 100x Acala?

yes dollars mean $ dollar value in sneks